docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -353,7 +353,14 @@ def depart_desc(self, node: Element) -> None:
     def visit_desc_signature(self, node: Element) -> None:
         self.in_literal += 1
         self.new_state()
-        self.add_text("<dt>")
+        ids = node.get("ids")
+        if ids:
+            if len(ids) == 1:
+                self.add_text(f"<dt id='{ids[0]}'>")
+            else:
+                raise ValueError(f"Expected 1 id, got {len(ids)} for {node}")
```

Comment:
I ran this on our entire codebase and there was never more than one id, I raise an exception here to make it easier to debug (you can drop sphinx into a debugger with `sphinx -P`)